src/scope.ml

```diff
@@ -120,6 +120,68 @@ type mode =
       carries the environment for variables that will be bound in the
       representation of the RHS. *)
 
+(** [get_params_implicitness ss env t] returns a list representing the
+    formal parameters of a parser term [t] *)
+let get_params_implicitness : sig_state -> env -> p_term -> bool list =
+  fun ss env t ->
+  match t.elt with
+  | P_Prod(args, _)                ->
+      List.map (fun x -> match x with (_,_,i) -> i) args
+  | P_Iden(_, FullyExplicit)       -> []
```

Comment:
Shouldn't this function be split in two distinct functions depending on its use:
1. In the symbol/definition/theorem commands, we want to know what are the implicit arguments of the declared symbol. In this case, only P_Prod is useful, and default is [].
2. When scoping a term.

Reply:
It was in two different functions before my last commit. I couldn't decide whether I should split it or not, but it's now back to two different functions :)